qiskit/pulse/schedule.py

```diff
@@ -61,9 +62,10 @@ def __init__(self, *schedules: Union[ScheduleComponent, Tuple[int, ScheduleCompo
         self._name = name
         self._duration = 0
 
+        # These attributes are populated by ``_mutable_insert``
         self._timeslots = {}
         self.__children = []
-
+        self._parameter_table = defaultdict(list)
```

Comment:
This pattern is often repeated, is there a `ParameterTable` class?

Reply:
Yes but unfortunately it doesn't work here. Maybe a community member could generalize it in the future.